i18n/fi.yaml

```diff
@@ -0,0 +1,94 @@
+# General
+- id: read_more
+  translation: "Lue enemmän…"
+
+- id: menu_label
+  translation: "Päävalikko"
+
+- id: sidemenu_title
+  translation: "Sivuvalikko"
+
+# Post meta
+- id: meta_lastmod
+  translation: "Viimeksi muutettu"
+
+- id: meta_translations
+  translation: "Käännökset"
+
+# Table of Contents
+- id: toc_title
+  translation: "Sisältöluettelo"
+
+# Post nav
+- id: post_nav_prev
+  translation: "Edellinen"
+
+- id: post_nav_next
+  translation: "Seuraava"
+
+# Authorbox
+- id: authorbox_name
+  translation: "Kirjoittaja {{ .Name }}"
+
+# Sidebar
+- id: sidebar_warning
+  translation: "VAROITUS"
+
+- id: sidebar_recommendation
+  translation: "Ystävällisesti aktivoi ainakin yksi Sivuvalikko widget."
```

Comment:
> Sivuvalikko

I'm not a native, so I could be wrong, but it seems like it's supposed to be different. It must be a translation of the word "sidebar". Right now it is the same as used in sidemenu_title. Sidemenu is not a whole sidebar thing, the names should be different, otherwise it will confuse the users.

Sivupalkki?

Reply:
I fixed it to "sidebar widget". It is more understandable in my opinion and this is used in some other translation (e.g. NL)